### ISSUE TITLE
Compatibility issue with the Dokan Multivendor plugin

### ISSUE THREAD

**Issue by upenkapse** [TycheSoftwares/custom-order-statuses-woocommerce#76]
When we assign a custom order status to order, it is not coming under the Orders section of the front-end vendor dashboard of the Dokan plugin.

As you can see in the below screenshots:
Admin orders page:-  https://prnt.sc/uev638
Dokan dashboard:-  https://prnt.sc/uev5i9

Ticket link: https://tychesoftwares.freshdesk.com/a/tickets/32837

**Comment by Chetna1510** [comment]
One client has asked for making this compatible with WC Marketplace: https://ru.wordpress.org/plugins/dc-woocommerce-multi-vendor/

**Comment by ashokrane** [comment]
@Chetna1510 @upenkapse I would suggest we remove this from the Lite version & put it in Pro. We don't want to give away vendor compatibility in the Lite version.

Let me know your thoughts.

**Comment by Chetna1510** [comment]
Yes, I agree with you @ashokrane We can inform the client that this compatibility has been added in premium version and we don't have any plan to include it in LITE

**Comment by ashokrane** [comment]
@Chetna1510 Yes, we can do that. A question: Have we added it in Premium version?

**Comment by Chetna1510** [comment]
No, @ashokrane We have a plan to add this compatibility in v2.5.0 . We will inform them once it is added in the PRO.

**Comment by ashokrane** [comment]
Ok, thanks @Chetna1510 .
